internal_transfers/database/README.md

```diff
@@ -0,0 +1,43 @@
+The system stores all data in a postgres database.
+This repo contains the docker image and the flyway migrations to run the database.
+
+## [Optional] Empty Local Instance of Postgres
+
+```sh
+docker run -e POSTGRES_PASSWORD=password -p 5432:5432 docker.io/postgres
+```
+
+## Perform Migration
+
+Prepare your environment (Database credentials)
+
+```shell
+cd internal_transfers/database
+cp .env.sample .env
+# populate .env with relevant fields
+source .env
+```
+
+Now, assuming environment variables are in place:
+
+```sh
+docker build --tag migration-image .
+# Remote DB Instance
+docker run --add-host=host.docker.internal:host-gateway --rm migration-image -url=jdbc:postgresql://$POSTGRES_HOST/$POSTGRES_DB -user=$POSTGRES_USER -password=$POSTGRES_PASSWORD migrate
+# Local DB Instance (from above)
+docker run --network host --add-host=localhost:host-gateway --rm migration-image -url=jdbc:postgresql://$POSTGRES_HOST/$POSTGRES_DB -user=$POSTGRES_USER -password=$POSTGRES_PASSWORD migrate
```

Comment:
`$POSTGRES_PASSWORD` and the password from the example above don't match in this case and result in a password authentication failed error.